src/app/asset-upload/asset-upload.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="asset-upload">
+    <mat-expansion-panel id="asset-upload-panel">
+        <mat-expansion-panel-header>
+            <mat-panel-title>
+                Asset Upload
+            </mat-panel-title>
+        </mat-expansion-panel-header>
+        <button mat-raised-button 
+            color="primary" 
+            type="button" 
+            (click)="fileInput.click()">
+            <span>Upload</span>
+            <input #fileInput 
+                type="file" 
+                (change)="onFileInput($event)" 
+                style="display:none;" 
```

Comment:
Whenever possible, it's preferable to define styling in the css (or scss) file.

https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns